navigation/presentation.html

```diff
@@ -18,8 +18,1169 @@
+        <section>
+          <!-- 12.0 slide -->
+          <section class="topic-name-slide">
+            <h4>Navigation lifecycle</h4>
+
+            <aside class="notes">
+              We looked at types of navigation and how to move between screens.
+              <br /><br />
+              An important question in this context is: what happens with some screen when we navigate away from it to other screen, or when we come back to it?
+              <br /><br />
+              How does a route find out that a user is leaving it or coming back to it?
+            </aside>
+          </section>
+          <!-- 12.1 slide -->
+          <section class="picture-slide">
+            <h4 class="slide-name">Navigation lifecycle</h4>
+            <img src="assets/images/navigation_lifecycle_1.png" alt="navigation lifecycle">
+
+            <aside class="notes">
+              Consider a stack navigator with screens A and B.
+              <br /><br />
+              After navigating to A, its componentDidMount is called.
+              <br /><br />
+              When pushing B, its componentDidMount is also called, but A remains mounted on the stack and its componentWillUnmount is therefore not called.
+              <br /><br />
+              When going back from B to A, componentWillUnmount of B is called, but componentDidMount of A is not because A remained mounted the whole time.
+            </aside>
+          </section>
+          <!-- 12.2 slide -->
+          <section class="picture-slide">
+            <h4 class="slide-name">Navigation lifecycle</h4>
+            <img src="assets/images/navigation_lifecycle_2.png" alt="navigation lifecycle">
+
+            <aside class="notes">
+              Consider a tab navigator with two tabs – tab ”One” and tab “Two”, where each tab is a stack navigator of two screens - A and B for the first tab, and C and D for the second.
+              <br /><br />
+              We start on the Screen A and navigate to Screen B. Then we use the tab bar to switch to the Screen C and navigate to Screen D.
+              <br /><br />
+              After this sequence of operations is done, all 4 of the screens are mounted!
+              <br /><br />
+              If you use the tab bar to switch back to the Screen A, you'll notice you'll be presented with the Screen B - the navigation state of the Screen A has been preserved!
+              <br /><br />
+              Now that we understand how React lifecycle methods work in React Navigation, let's answer the question we asked at the beginning: "How do we find out that a user is leaving it or coming back to it?”
+              <br /><br />
+              ===> next slide
+            </aside>
+          </section>
+        </section>
+        <!-- 13 slide -->
+        <section class="text-and-conding-slide">
+          <h4 class="slide-name">Navigation events</h4>
+          <div class="center-align">
+            <p>
+              core events:
```

Comment:
I have the p elements displayed halfway, only the top part. The mobile version is fine

Reply:
fixed